packages/generator-open-wc/generators/starter-app/templates/static/src/index.js

```diff
@@ -0,0 +1,5 @@
+import loadPolyfills from '@open-wc/polyfills-loader';
+
+loadPolyfills().then(() => {
```

Comment:
this file automatically gets generated when you call the building-webpack generator?

better to reuse existing generators